dotcom-rendering/src/web/lib/renderElement.tsx

```diff
@@ -183,19 +183,16 @@ export const renderElement = ({
 				</Island>
 			);
 		case 'model.dotcomrendering.pageElements.CalloutBlockElementV2':
-			if (
-				switches.callouts &&
-				abTests?.calloutElementsVariant === 'variant'
-			) {
-				return (
-					<Island deferUntil="visible">
-						<CalloutBlockComponent
-							callout={element}
-							format={format}
-						/>
-					</Island>
-				);
-			}
+			// if (
+			// 	switches.callouts &&
+			// 	abTests?.calloutElementsVariant === 'variant'
+			// ) {
```

Comment:
Is this intended to remove the check?

Reply:
Oops thanks for spotting that Ioanna! Must have accidentally committed that when testing stuff. Good spot.